lesson-34/index.ts

```diff
@@ -0,0 +1,100 @@
+interface IFaculties {
+    id: number;
+    faculty: string;
+    subjects: string[];
+    countStudents: number
+}
+
+
+
+
+1. const faculties = [
```

Comment:
And here you need to define type of this array of faculties, like this: 
`const faculties: IFaculties[] = [...]`